test/unit_tests/arkode/CXX_serial/ark_test_dahlquist_mri.cpp

```diff
@@ -418,10 +414,11 @@ int run_tests(MRISTEP_METHOD_TYPE type, sunrealtype t0, int nsteps,
 
     cout << "\nComparing Solver Statistics:\n";
 
+    int nstages_evaled = nstages_stored - stiffly_accurate;
```

Comment:
I'm not sure what "evaled" means -- can this be changed to something clearer (e.g., "evaluated")?